lib/erb/formatter/version.rb

```diff
@@ -3,5 +3,5 @@
 require 'erb'
 
 class ERB::Formatter
-  VERSION = "0.3.0"
+  VERSION = '0.3.1'
```

Comment:
Can you skip this change?
We'll bump the version prior to the release and will pick whether to update the major/minor/patch depending on the newly merged PRs.

Reply:
Done